bindata/network/openshift-sdn/004-multitenant.yaml

```diff
@@ -126,4 +128,19 @@ metadata:
 netid: 1
 netname: openshift-service-catalog-controller-manager
 
+---
+apiVersion: network.openshift.io/v1
+kind: NetNamespace
+metadata:
+  name: openshift-authentication
+netid: 1
+netname: openshift-authentication
+
+---
+apiVersion: network.openshift.io/v1
+kind: NetNamespace
+metadata:
+  name: openshift-authentication-operator
+netid: 1
+netname: openshift-authentication-operator
```

Comment:
when do we need such a rule? For every cross-namespace communication?

Reply:
Isn't it questionable to have these rules here in the network operator? The network operator does not know all software running on the cluster. Compare with prometheus monitor manifests, which were moved into the monitored components repos.

Reply:
I remember having talked about the idea of components putting annotations on their Namespaces but I don't remember why we did it this way instead. I think we wanted to avoid having other components have to worry about openshift-sdn multitenant mode because it's more-or-less deprecated anyway. (It's difficult to enable in openshift-sdn in 4.x, and it won't exist at all in ovn-kubernetes.)

Reply:
> when do we need such a rule? For every cross-namespace communication?

several of the more exciting namespaces (eg, openshift-monitoring, openshift-kube-apiserver) are made global so that anyone can talk to them and they can talk to anyone, and that covers almost all of the interactions.

Reply:
ack